component/prometheus/exporter/windows/config.go

```diff
@@ -0,0 +1,190 @@
+package windows
+
+import (
+	windows_integration "github.com/grafana/agent/pkg/integrations/windows_exporter"
+)
+
+// DefaultArguments holds non-zero default options for Arguments when it is
+// unmarshaled from YAML.
+//
+// Some defaults are populated from init functions in the github.com/grafana/agent/pkg/integrations/node_exporter package.
+var DefaultArguments = Arguments{
+	EnabledCollectors: "cpu,cs,logical_disk,net,os,service,system",
+}
+
+// Arguments is used for controlling for this exporter.
+type Arguments struct {
+	// Collectors to mark as enabled
+	EnabledCollectors string `river:"enabled_collectors,attr,optional"`
+
+	// Collector-specific config options
+	Exchange    ExchangeConfig    `river:"exchange,block,optional"`
+	IIS         IISConfig         `river:"iis,block,optional"`
+	TextFile    TextFileConfig    `river:"text_file,block,optional"`
+	SMTP        SMTPConfig        `river:"smtp,block,optional"`
+	Service     ServiceConfig     `river:"service,block,optional"`
+	Process     ProcessConfig     `river:"process,block,optional"`
+	Network     NetworkConfig     `river:"network,block,optional"`
+	MSSQL       MSSQLConfig       `river:"mssql,block,optional"`
+	MSMQ        MSMQConfig        `river:"msmq,block,optional"`
+	LogicalDisk LogicalDiskConfig `river:"logical_disk,block,optional"`
+}
+
+// UnmarshalRiver implements River unmarshalling for Config.
+func (a *Arguments) UnmarshalRiver(f func(interface{}) error) error {
+	*a = DefaultArguments
+
+	type args Arguments
+	return f((*args)(a))
+}
+
+// Convert converts the component's Arguments to the integration's Config.
+func (a *Arguments) Convert() *windows_integration.Config {
+	return &windows_integration.Config{
+		EnabledCollectors: a.EnabledCollectors,
+		Exchange:          a.Exchange.Convert(),
+		IIS:               a.IIS.Convert(),
+		TextFile:          a.TextFile.Convert(),
+		SMTP:              a.SMTP.Convert(),
+		Service:           a.Service.Convert(),
+		Process:           a.Process.Convert(),
+		Network:           a.Network.Convert(),
+		MSSQL:             a.MSSQL.Convert(),
+		MSMQ:              a.MSMQ.Convert(),
+		LogicalDisk:       a.LogicalDisk.Convert(),
+	}
+}
+
+// ExchangeConfig handles settings for the windows_exporter Exchange collector
+type ExchangeConfig struct {
+	EnabledList string `river:"enabled_list,attr,optional"`
```

Comment:
This is another one where there's an opportunity to make it a []string on the Flow side since it's supposed to be a list of strings anyway.